packages/iov-core/README.md

```diff
@@ -186,14 +185,11 @@ Query the testnet for some existing genesis accounts:
 ```ts
 // this is pulled from the genesis account
 import { Address } from "@iov/bcp-types"
-const bert = "E28AE9A6EB94FC88B73EB7CBD6B87BF93EB9BEF0" as Address;
-const faucet = await connection.getAccount({ address: bert });
-console.log(faucet);
-console.log(faucet.data[0])
 
-// you can also query by registered name
-const byName = await connection.getAccount({ name: "bert" });
-console.log(byName.data[0])
+const bert = "E28AE9A6EB94FC88B73EB7CBD6B87BF93EB9BEF0" as Address;
```

Comment:
Could you use the https://iov-one.github.io/token-finder/ to convert the old hex address format into bech32 using the "tiov" prefix?

Reply:
I've updated this, but the account is not found.

Reply:
Yeah, no problem, thanks